public/themes/default/pages/profile.html

```diff
@@ -24,10 +24,29 @@ <h1>
     </p>
 
     <p>
-      {{ profileUser.name }} joined {{ profileUser.created_at | year }} and
-      created {{ profile.projects.length }} projects since then!
+      {% if profile.ownProfile %}You{% else %}{{ profileUser.name }}{% endif %}
+      joined {{ profileUser.created_at | year }} and created {{
+      profile.projects.length }} projects since then!
     </p>
 
+    {% if profile.ownProfile %}
+    <section>
+      <h2>Login methods</h2>
+
+      <ul class="login-methods">
+        {% for service in profile.services %}
+        <li>{{ service }}</li>
```

Comment:
...just because users should be able to say "no" at a later date, this probably needs a "remove" option too. And of course folks should not be able to remove the only auth method.